src/rapid_pro_to_engagement_db/rapid_pro_to_engagement_db.py

```diff
@@ -258,7 +259,7 @@ def sync_rapid_pro_to_engagement_db(rapid_pro, engagement_db, uuid_table, rapid_
                     cache.set_latest_run_timestamp(flow_id, run.modified_on)
                 continue
             contact = contacts_lut[run.contact.uuid]
-            assert len(contact.urns) == 1, len(contact.urns)
+            # assert len(contact.urns) == 1, len(contact.urns)
```

Comment:
Thanks for catching this. I needed to comment this out to run the test pipeline due to contacts in the test Rapid Pro workspace having 2 urns. I've reverted this here for now so it can be fixed when the support for whatsapp merges.